resources/puppetlabs/lein-ezbake/template/foss/ext/redhat/init.erb

```diff
@@ -59,44 +59,8 @@ find_my_pid() {
 }
 
 start() {
-    # call status here and figure out current state
-    rh_status_q
-    [ -x $JAVA_BIN ] || exit 5
-    [ -e $config ] || exit 6
```

Comment:
We can probably stand to lose the `JAVA_BIN` checking but do we want to lose the `$config` test? Does TK handle that now?